contracts/shiden34/lib.rs

```diff
@@ -67,9 +62,44 @@ pub mod shiden34 {
         });
     }
 
-    impl payable_mint_pkg::impls::payable_mint::payable_mint::Internal for Shiden34 {}
+    impl payable_mint::Internal for Shiden34 {}
+
     impl payable_mint::PayableMintImpl for Shiden34 {}
```

Comment:
Is this needed?

Reply:
yes, atm, otherwise the code won't compile